examples/Azure_IoT_Central_ESP32/Azure_IoT_PnP_Template.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/*
+ * Azure_IoT_PnP_Template.cpp implements the the Azure Plug-and-Play template
+ * specific for the Espressif ESP32 Azure IoT Kit board.
```

Comment:
This one's the generic ESP32 right?
```suggestion
 * specific for the Espressif ESP32 board.
```

Reply:
Ha. Thanks.